### PR TITLE
1.9.1 common changes

### PR DESCRIPTION
**Pull request by boyamurthy**
Updating 1.9.1 patch release documentation 

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/1109 |

# Checklist:

- [X] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by shanmydell — DISMISSED** [review]

File: content/docs/csidriver/installation/offline/_index.md [hunk 1]
```diff
@@ -83,21 +83,21 @@ bash scripts/csm-offline-bundle.sh -c
 *
 * Pulling and saving container images
 
-   dellemc/csi-isilon:v2.8.0
+   dellemc/csi-isilon:v2.8.1
    dellemc/csi-metadata-retriever:v1.6.0
```
Comment (AkshaySainiDell) [inline content/docs/csidriver/installation/offline/_index.md:87]:
```suggestion
   dellemc/csi-isilon:v2.9.1
   dellemc/csi-metadata-retriever:v1.6.1
```

**Review by AkshaySainiDell — CHANGES_REQUESTED** [review]

File: content/docs/csidriver/troubleshooting/powerstore.md [hunk 2]
```diff
@@ -12,4 +12,11 @@ description: Troubleshooting PowerStore Driver
...
+| If two separate networks are configured for ISCSI and NVMeTCP, the driver may encounter difficulty identifying the second network (e.g., NVMeTCP). | This is a known issue, and the workaround involves creating a single network on the array to serve both ISCSI and NVMeTCP purposes. |
+|Driver install or upgrade fails because of an incompatible Kubernetes version, even though the version seems to be within the range of compatibility. For example: Error: UPGRADE FAILED: chart requires kubeVersion: >= 1.22.0 < 1.25.0 which is incompatible with Kubernetes V1.22.11-mirantis-1 | If you are using an extended Kubernetes version, please see the [helm Chart](https://github.com/dell/helm-charts/blob/main/charts/csi-powerstore/Chart.yaml) and use the alternate kubeVersion check that is provided in the comments. Please note that this is not meant to be used to enable the use of pre-release alpha and beta versions, which is not supported.|
```
Comment (AkshaySainiDell) [inline content/docs/csidriver/troubleshooting/powerstore.md:17]: This point is duplicated.

File: content/docs/authorization/release/_index.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ Description: >
   Dell Container Storage Modules (CSM) release notes for authorization
 ---
 
-## Release Notes - CSM Authorization 1.8.0
+## Release Notes - CSM Authorization 1.8.1
```
[the remark this answers is not in the document]
Comment (boyamurthy) [reply content/docs/authorization/release/_index.md]: updated .

File: content/docs/csidriver/installation/offline/_index.md (hunk 1 above)
Comment (boyamurthy) [reply content/docs/csidriver/installation/offline/_index.md]: updated .

File: content/docs/deployment/csminstallationwizard/_index.md [hunk 4]
```diff
@@ -97,7 +97,7 @@ The [Dell Container Storage Modules Installation Wizard](./src/index.html) is a
 
     ```terminal
     helm install <release-name> dell/container-storage-modules -n <namespace> --version <container-storage-module chart-version> -f <values.yaml location>
-    Example: helm install powerstore dell/container-storage-modules -n csi-powerstore --version 1.1.0 -f values.yaml
+    Example: helm install powerstore dell/container-storage-modules -n csi-powerstore --version 1.2.0 -f values.yaml
```
Comment (AkshaySainiDell) [inline content/docs/deployment/csminstallationwizard/_index.md:100]:
```suggestion
    Example: helm install powerstore dell/container-storage-modules -n csi-powerstore --version 1.2.1 -f values.yaml
```

**Review by AkshaySainiDell — CHANGES_REQUESTED** [review]

File: content/docs/csidriver/troubleshooting/powerstore.md (hunk 2 above)
Comment (boyamurthy) [reply content/docs/csidriver/troubleshooting/powerstore.md]: fixed .

File: content/docs/deployment/csminstallationwizard/_index.md (hunk 4 above)
Comment (boyamurthy) [reply content/docs/deployment/csminstallationwizard/_index.md]: fixed .

**Review by mareksuski-dell — APPROVED** [review]
LGTM

**Review by AkshaySainiDell — APPROVED** [review]

**Pull request merged**